### PR TITLE
playbook splitting

### PR DESCRIPTION
**Pull request by vvikramb**
playbook splitting based on category.

### CODE REVIEW

File: juniper_official/Interfaces/interface-out-errors-netconf.rule [hunk 1]
```diff
@@ -104,6 +104,10 @@ healthbot {
             rule-properties {
```
[the remark this answers is not in the document]
Comment (vvikramb) [reply juniper_official/Interfaces/interface-out-errors-netconf.rule:104]: fixed.

File: juniper_official/Chassis/chassis-alarms.rule [hunk 2]
```diff
@@ -89,6 +89,10 @@
             rule-properties {
                 version 2;
                 contributor juniper;
+                category basic;
+                is-scaling-rule {
```
Comment (gvarao) [inline juniper_official/Chassis/chassis-alarms.rule:93]: This is wrong. Scaling required only for suppose to be scaling rules. Revisit all rules and modify as shown below;

e.g.
Processes-"Fields:[ daemon pid ]; Directly impacted by number of processes running in each network device. Average 150+ processes in junos"
Interfaces
BGP
OSPF neighbors
ISIS adjacencies
L3VPNs
EVPNs
LSPs
LLDP
RIB Route-summary
FIB
SFLOW
RSVP sessions
Subscriber services - Check

**Review by gvarao — CHANGES_REQUESTED** [review]

File: juniper_official/Interfaces/interface-flaps-iagent.rule [hunk 3]
```diff
@@ -93,6 +93,10 @@ healthbot {
             rule-properties {
```
[the remark this answers is not in the document]
Comment (vvikramb) [reply juniper_official/Interfaces/interface-flaps-iagent.rule:93]: fixed.

File: juniper_official/Chassis/chassis-alarms.rule (hunk 2 above)
Comment (vvikramb) [reply juniper_official/Chassis/chassis-alarms.rule]: Made all the changes.

File: juniper_official/Chassis/netsvc.rule [hunk 4]
```diff
@@ -9,7 +9,7 @@ healthbot {
                 }
             }
             trigger netsvc-sync {
-		frequency 1offset;
+		        frequency 1offset;
```
Comment (gvarao) [inline juniper_official/Chassis/netsvc.rule:12]: Correct indentations

File: juniper_official/System/system-storage-capacity.rule [hunk 5]
```diff
@@ -91,6 +91,10 @@ healthbot {
             rule-properties {
                 version 2;
                 contributor juniper;
+                category advanced;
+                is-scaling-rule {
```
Comment (gvarao) [inline juniper_official/System/system-storage-capacity.rule:95]: System storage is not scale rule as juniper device will have hardly <10 mount paths

File: juniper_official/System/system-storage.rule [hunk 6]
```diff
@@ -169,6 +169,10 @@ healthbot {
             rule-properties {
                 version 2;
                 contributor juniper;
+                category basic;
+                is-scaling-rule {
```
Comment (gvarao) [inline juniper_official/System/system-storage.rule:173]: Not scale rule as juniper device have less mount paths

**Review by gvarao — CHANGES_REQUESTED** [review]

File: juniper_official/Chassis/netsvc.rule (hunk 4 above)
Comment (vvikramb) [reply juniper_official/Chassis/netsvc.rule]: Corrected.

File: juniper_official/System/system-storage.rule (hunk 6 above)
Comment (vvikramb) [reply juniper_official/System/system-storage.rule]: Removed scaling.

File: juniper_official/System/system-storage-capacity.rule (hunk 5 above)
Comment (vvikramb) [reply juniper_official/System/system-storage-capacity.rule]: Removed scaling.

**Review by gvarao — APPROVED** [review]

**Pull request merged**